src/components/book/InputBook.js

```diff
@@ -0,0 +1,17 @@
+import React, { PureComponent } from 'react';
+
+class InputBook extends PureComponent {
+  render() {
```

Comment:
- [ **OPTIONAL** ] What do you think if you you use `function-based` on all your applications components for best practices.